CRITICAL - 2020-10-01 19:21:29 --> Undefined property: App\Controllers\Home::$model
#0 C:\xampp\htdocs\tpq\app\Controllers\Home.php(19): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 19, Array)
#1 C:\xampp\htdocs\tpq\system\CodeIgniter.php(918): App\Controllers\Home->index()
#2 C:\xampp\htdocs\tpq\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#3 C:\xampp\htdocs\tpq\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\tpq\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\tpq\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2020-10-01 19:23:08 --> Constant expression contains invalid operations
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-10-01 19:45:58 --> Unknown column 'id_ust' in 'field list'
#0 C:\xampp\htdocs\tpq\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `id_ust`...')
#1 C:\xampp\htdocs\tpq\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id_ust`...')
#2 C:\xampp\htdocs\tpq\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id_ust`...')
#3 C:\xampp\htdocs\tpq\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `id_ust`...', Array, false)
#4 C:\xampp\htdocs\tpq\app\Models\AbsensiModel.php(46): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tpq\app\Controllers\Home.php(35): App\Models\AbsensiModel->tulisAbsen()
#6 C:\xampp\htdocs\tpq\system\CodeIgniter.php(918): App\Controllers\Home->tulisAbsen()
#7 C:\xampp\htdocs\tpq\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\tpq\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tpq\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tpq\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2020-10-01 19:45:59 --> Unknown column 'id_ust' in 'field list'
#0 C:\xampp\htdocs\tpq\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `id_ust`...')
#1 C:\xampp\htdocs\tpq\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id_ust`...')
#2 C:\xampp\htdocs\tpq\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id_ust`...')
#3 C:\xampp\htdocs\tpq\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `id_ust`...', Array, false)
#4 C:\xampp\htdocs\tpq\app\Models\AbsensiModel.php(46): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tpq\app\Controllers\Home.php(35): App\Models\AbsensiModel->tulisAbsen()
#6 C:\xampp\htdocs\tpq\system\CodeIgniter.php(918): App\Controllers\Home->tulisAbsen()
#7 C:\xampp\htdocs\tpq\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\tpq\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tpq\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tpq\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2020-10-01 19:51:46 --> Undefined variable: aata
#0 C:\xampp\htdocs\tpq\app\Models\AbsensiModel.php(58): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 58, Array)
#1 C:\xampp\htdocs\tpq\app\Controllers\Home.php(35): App\Models\AbsensiModel->tulisAbsen()
#2 C:\xampp\htdocs\tpq\system\CodeIgniter.php(918): App\Controllers\Home->tulisAbsen()
#3 C:\xampp\htdocs\tpq\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#4 C:\xampp\htdocs\tpq\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\tpq\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\tpq\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2020-10-01 19:52:17 --> Trying to get property 'id' of non-object
#0 C:\xampp\htdocs\tpq\app\Models\AbsensiModel.php(51): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 51, Array)
#1 C:\xampp\htdocs\tpq\app\Controllers\Home.php(35): App\Models\AbsensiModel->tulisAbsen()
#2 C:\xampp\htdocs\tpq\system\CodeIgniter.php(918): App\Controllers\Home->tulisAbsen()
#3 C:\xampp\htdocs\tpq\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#4 C:\xampp\htdocs\tpq\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\tpq\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\tpq\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2020-10-01 20:09:17 --> Call to undefined function DateTime()
#0 C:\xampp\htdocs\tpq\system\View\View.php(246): include()
#1 C:\xampp\htdocs\tpq\system\Common.php(1119): CodeIgniter\View\View->render('pages/home', Array, true)
#2 C:\xampp\htdocs\tpq\app\Controllers\Home.php(24): view('pages/home', Array)
#3 C:\xampp\htdocs\tpq\system\CodeIgniter.php(918): App\Controllers\Home->index()
#4 C:\xampp\htdocs\tpq\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\tpq\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\tpq\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\tpq\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-10-01 20:16:25 --> Unable to connect to the database.
#0 C:\xampp\htdocs\tpq\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\tpq\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `id_ust`...', Array, false)
#2 C:\xampp\htdocs\tpq\app\Models\AbsensiModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\tpq\app\Controllers\Home.php(19): App\Models\AbsensiModel->getAbsensi()
#4 C:\xampp\htdocs\tpq\system\CodeIgniter.php(918): App\Controllers\Home->index()
#5 C:\xampp\htdocs\tpq\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\tpq\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\tpq\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\tpq\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-10-01 20:16:26 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\tpq\app\Views\errors\html\error_exception.php:100
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 100, Array)
#1 C:\xampp\htdocs\tpq\app\Views\errors\html\error_exception.php(100): print_r(Object(App\Controllers\Home), true)
#2 C:\xampp\htdocs\tpq\system\Debug\Exceptions.php(308): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\tpq\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-10-01 20:27:41 --> Namespace declaration statement has to be the very first statement or after any declare call in the script
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
